TileRarities.lua

```diff
@@ -0,0 +1,131 @@
+--[[
+    The data for the different chances and multiplier values can be found on Data:Tile_Chances
+]]--
+local p = {}
```

Comment:
add another table for helper functions:

```lua
local p = {}
local h = {} -- for helper functions
```

(note: this is needed so Comment A can happen)